### DISCUSSION TITLE
Log file Build 20240613

### DISCUSSION THREAD

**Discussion by dineshjk** [WinEdt-Team/WinEdt-Support#70] [1.1 General Help and Discussion]
Earlier versions of WinEdt had a facility that when the console stops with an error, pressing X opened the log file. If I remember ccorrectly, we had to set it from execution modes dialogue box. But I am lost. i am not able to do it now. Also, Ctrl+L opened the log file of the corresponding to the current project. Now it runs LaTeX. How can I restore these two facilities without disturbing other good things.

**Comment by Alex-Simonic** [comment]
On the first page of the Execution Modes click on the accessory you are using (e.g. PDFTeXify, PDFLaTeX, XeLaTeX, etc...) and enable the option "Open Error Dialog". The same can be achieved manually with the Ctrl+E shortcut. 

As for Ctrl+L: all shortcuts are customizable. In the MainMenu.ini (Options Interface) you will find the definition of the item "Log File" in the Document menu.

```
  ITEM="LOG_File"
    CAPTION="&LOG File"
    IMAGE="LOG"
    MACRO="UpdateFile('%E');Open('%E',1);"
    REQ_FILTER="%E"
```

You can add 

`      SHORTCUT="16460::Ctrl+L"
`

**Comment by dineshjk** [comment]
Thanks Alex,

However, I am still missing one important aspect. Earlier, when we pressed
x after error, the control transferred to the .tex file at the error
location. Now, it is not transferring the control to the .tex file.

Dinesh
**********
Dinesh Karia : Free Bird
**********
-- 



*"It's God's responsibility to forgive the terrorist organizations  We
arrange the meeting between them and god."                  - Indian Armed
Forces -                                                           *

Interested in discussion on Mathematics? Click Me:
<http://groups.yahoo.com/group/ugcnetmath>

**Reply by Alex-Simonic** [reply]
I have no idea which previous version you are referring to and what you mean by pressing X. Nothing about the execution flow has changed in any recent version of WinEdt. Perhaps you typed e (not x)?
